Firestore/Source/Remote/FSTRemoteEvent.mm

```diff
@@ -212,15 +217,22 @@ + (instancetype)changeWithDocuments:(NSArray<FSTMaybeDocument *> *)docs
 }
 
 + (instancetype)changeWithMapping:(FSTTargetMapping *)mapping
-                  snapshotVersion:(FSTSnapshotVersion *)snapshotVersion
+                  snapshotVersion:(SnapshotVersion)snapshotVersion
               currentStatusUpdate:(FSTCurrentStatusUpdate)currentStatusUpdate {
   FSTTargetChange *change = [[FSTTargetChange alloc] init];
   change.mapping = mapping;
-  change.snapshotVersion = snapshotVersion;
+  change->_snapshotVersion = std::move(snapshotVersion);
   change.currentStatusUpdate = currentStatusUpdate;
   return change;
 }
 
++ (instancetype)changeWithSnapshotVersion:
```

Comment:
Better to just make this an initializer:

```objc
- (instancetype)initWithSnapshotVersion:(SnapshotVersion)snapshotVersion {
  if (self = [super init]) {
    _snapshotVersion = std::move(snapshotVersion);
  }
  return self;
}
```

Reply:
done